### ISSUE TITLE
What is the structural block diagram of LightningPose? Are the semi-supervised network and the temporal context network parallel?

### ISSUE THREAD

**Issue by wyclearnpy** [paninski-lab/lightning-pose#225]

**Comment by themattinthehatt** [comment]
@wyclearnpy the baseline supervised model is just a backbone (default: resnet-50) with a transpose convolution head that maps the backbone features to heatmaps. The TCN changes the architecture to look like this:
![Screenshot from 2024-12-02 14-52-48](https://github.com/user-attachments/assets/6d794c3a-f8c1-4523-bf49-31cd98733802)

This is from figure 2 of the paper: https://www.nature.com/articles/s41592-024-02319-1

The unsupervised losses do not depend on a particular architecture, they just provide additional loss functions to minimize during training. As such, the unsupervised losses can be used with either the baseline (non-context) model or the TCN model.

Does that answer your question?
